desktop/lib/messagebundles/es.po

```diff
@@ -2441,11 +2433,11 @@ msgstr ""
 
 #: /frostwire/desktop/src/com/limegroup/gnutella/gui/GUIMediator.java:113
 msgid "&Search"
-msgstr "Bu&scar"
+msgstr "&Buscar"
```

Comment:
I'm not too sure of this, it might work though...
![not-impressed](https://cloud.githubusercontent.com/assets/163977/12077193/07e09c4c-b1a5-11e5-907c-73a14e9e293e.jpg)